lib/std/x/os/net.zig

```diff
@@ -20,7 +20,7 @@ pub fn resolveScopeID(name: []const u8) !u32 {
     if (comptime @hasDecl(os, "IFNAMESIZE")) {
         if (name.len >= os.IFNAMESIZE - 1) return error.NameTooLong;
 
-        if (comptime builtin.os.tag == .windows) {
+        if (comptime native_os.tag == .windows) {
```

Comment:
This comptime and the one on line 20 are redundant

Reply:
Thanks! I'll remove the `comptime` :smile:.